templates/python/LocalPubSub/test_localpubsub/test_publisher.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+from localpubsub import publisher
+from awsiot.greengrasscoreipc.model import PublishMessage, BinaryMessage
+
+@pytest.fixture
+def ipc_client(mocker):
+    mocker.patch("awsiot.greengrasscoreipc.clientv2.GreengrassCoreIPCClientV2")
+    mocker.patch("awsiot.greengrasscoreipc.connect")
+    from awsiot.greengrasscoreipc.clientv2 import GreengrassCoreIPCClientV2
+
+    client = GreengrassCoreIPCClientV2()
+    return client
+
+
+def test_publish_message_N_times(ipc_client):
+    publisher.publish_message_N_times(ipc_client, "topic", "World")
+    assert ipc_client.publish_to_topic.call_count == 10
```

Comment:
Don't assert the call count, check it gets called with the expected args at least.